README.md

```diff
@@ -32,11 +32,14 @@ Also, if you want to request a particular package, you may suggest it in an issu
 
 If your device running Android OS with version 7 and higher, it is highly recommended to check whether your Termux installation uses our new repository with packages compiled specially for higher Android API levels (24+).
 
-Execute command `termux-info` or look into file `$PREFIX/etc/apt/sources.list`. There should be a line containing following URL:
-```
-https://dl.bintray.com/termux/termux-packages-24/
+Execute any one of these commands:
+```bash
+$ termux-info | grep -o 'https://dl.bintray.com/termux/termux-packages-[(0-9)]*' | cut -d- -f3'
+
```

Comment:
Commands should be **generic as possible**.

Reply:
It's already fair enough then making user find some line of text in a bunch of lines contained in a file or from output of a command

Reply:
Looks like you never tried to copy-paste **long** lines from Github.
And if line was meant to be copy-pasted, then why you included `$` at begin ?